CMakeLists.txt

```diff
@@ -329,33 +114,6 @@ configure_package_config_file(
   INSTALL_DESTINATION "${config_install_dir}"
 )
 
-# Targets:
-#   * <prefix>/lib/libjaegertracing.a
-#   * <prefix>/lib/libjaegertracing.so
-#   * header location after install: <prefix>/include/jaegertracing/Tracer.h
-#   * headers can be included by C++ code `#include <jaegertracing/Tracer.h>`
-install(
-  TARGETS jaegertracing jaegertracing-static
-  EXPORT "${TARGETS_EXPORT_NAME}"
-  LIBRARY DESTINATION "lib"
-  ARCHIVE DESTINATION "lib"
-  RUNTIME DESTINATION "bin"
-  INCLUDES DESTINATION "${include_install_dir}"
-)
```

Comment:
Why remove this?

Reply:
Move, not remove. Installation of the library target is in the library's own `CMakeLists.txt` in `src/jaegertracing`.

Reply:
OK